editors/vim/syntax/fluent.vim

```diff
@@ -0,0 +1,26 @@
+if exists("b:current_syntax")
+    finish
+endif
+
+syntax match fluentComment "\v#.*$"
+syntax match fluentIdentifier "^\v[a-zA-Z-]+" nextgroup=fluentDelimiter
+syntax match fluentDelimiter "\s*=\s*" contained skipnl nextgroup=fluentPattern
+syntax region fluentPattern contained start="" end="\v^(( )@!|( +[\.\[\*\}])@=)" contains=fluentPlaceableExpression
+syntax match fluentAttribute "\v\.[a-zA-Z-]+" nextgroup=fluentDelimiter
+syntax region fluentPlaceableExpression contained matchgroup=fluentPlaceableBraces start=+{+ end=+}+ contains=@fluentExpression
```

Comment:
Can this be simply called `fluentPlaceable`? Also, what's `fluentPlaceableBrace` and where is it used/defined?